firstvoices/backend/serializers/media_serializers.py

```diff
@@ -262,3 +262,18 @@ class RelatedMediaSerializerMixin(metaclass=serializers.SerializerMetaclass):
 
     class Meta:
         fields = ("related_audio", "related_images", "related_videos")
+
+
+class RelatedAudioMinimalSerializer(serializers.ModelSerializer):
+    original = MediaFileUploadSerializer(read_only=True)
```

Comment:
A read-only MediaFileUploadSerializer is just a MediaFileSerializer -- might as well use that for clarity.